dist/tools/externc/check.sh

```diff
@@ -18,7 +18,8 @@ cd $RIOTBASE
 ROOT=$(git rev-parse --show-toplevel)
 EXIT_CODE=0
 
-FILES=$(FILEREGEX='\.h$' changed_files)
+EXCLUDE='^(dist/tools/)'
+FILES=$(FILEREGEX='\.h$' EXCLUDE=${EXCLUDE} changed_files)
```

Comment:
for some reason now this check is also run for `vendor/` files that have not been checked before

Reply:
Hmm, it's because it overwrites the predefined EXCLUDE:

https://github.com/RIOT-OS/RIOT/blob/0427e28f170002b63130afd0a0bd26c6f68324b9/dist/tools/ci/changed_files.sh#L11

And it should append to it. Sorry for that.

Reply:
Ah no, in fact the `boards/common/msba2/tools/src` should be replaced by `dist/tools/lpc2k_pgm/src`

Reply:
Should be fixed by #15552